isolate_example/lib/page_one.dart

```diff
@@ -84,7 +178,10 @@ class SmoothAnimationWidgetState extends State<SmoothAnimationWidget>
             decoration: BoxDecoration(
               gradient: LinearGradient(
                 begin: Alignment.topLeft,
-                colors: [Colors.blueAccent, Colors.redAccent],
+                colors: <Color>[
+                  Colors.blueAccent,
+                  Colors.redAccent
+                ], // whitish to gray
```

Comment:
What does "whitish to gray" mean in this context? Is that a leftover from a previous pair of colors?